packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/index.tsx

```diff
@@ -112,39 +117,59 @@ export const PipelineHeader: React.FunctionComponent<PipelineHeaderProps> = ({
   showRunButton,
   showExportButton,
   showExplainButton,
+  onHideAIInputClick,
   onToggleOptions,
+  isAIInputVisible = false,
   isOptionsVisible,
   isOpenPipelineVisible,
 }) => {
+  const enableAIExperience = usePreference('enableAIExperience', React);
+
   return (
-    <div className={containerStyles} data-testid="pipeline-header">
-      {isOpenPipelineVisible && (
-        <div
-          data-testid="saved-pipelines-popover"
-          className={pipelineTextAndOpenStyles}
-        >
-          <Body weight="medium">Pipeline</Body>
-          <SavedPipelinesButton></SavedPipelinesButton>
+    <div>
+      <div className={containerStyles} data-testid="pipeline-header">
+        {isOpenPipelineVisible && (
+          <div
+            data-testid="saved-pipelines-popover"
+            className={pipelineTextAndOpenStyles}
+          >
+            <Body weight="medium">Pipeline</Body>
+            <SavedPipelinesButton></SavedPipelinesButton>
+          </div>
+        )}
+        <div className={pipelineStagesStyles}>
+          <PipelineStages />
+        </div>
+        <div className={pipelineActionStyles}>
+          <PipelineActions
+            onToggleOptions={onToggleOptions}
+            isOptionsVisible={isOptionsVisible}
+            showRunButton={showRunButton}
+            showExportButton={showExportButton}
+            showExplainButton={showExplainButton}
+          />
         </div>
-      )}
-      <div className={pipelineStagesStyles}>
-        <PipelineStages />
       </div>
-      <div className={pipelineActionStyles}>
-        <PipelineActions
-          onToggleOptions={onToggleOptions}
-          isOptionsVisible={isOptionsVisible}
-          showRunButton={showRunButton}
-          showExportButton={showExportButton}
-          showExplainButton={showExplainButton}
+      {enableAIExperience && (
+        <PipelineAI
+          onClose={() => {
+            onHideAIInputClick?.();
+          }}
+          show={isAIInputVisible}
         />
-      </div>
+      )}
```

Comment:
Is this by design that this is added above the options and not below? Why is it different from query bar?

![image](https://github.com/mongodb-js/compass/assets/5036933/694f1481-39b6-4fbf-8efb-1bb7d95517d8)

Reply:
Nope! Thanks for catching. Pushed up an update:
<img width="1049" alt="Screenshot 2023-08-18 at 11 24 39 AM" src="https://github.com/mongodb-js/compass/assets/1791149/b03e011a-8857-4986-b105-f0e59c403e92">